### DISCUSSION TITLE
Two questions

### DISCUSSION THREAD

**Discussion by 4JX** [fltk-rs/fltk-rs#1051] [Q&A] [answered]
- Is it possible to enable some form of anti-aliasing on the UI windows? Curved edges and frames look a bit jagged.
- What's the closest fltk equivalent to this type of button? (Where one deactivates when the other is selected)
![Image](https://i.ibb.co/82RmRyL/Indasearchofthebutton.gif)

**Comment by MoAlyousef** [comment, marked as answer]
1- Currently FLTK uses X11 on Linux, which doesn't have anti-aliasing. There's work to use Cairo graphics there. On Windows and MacOS, it's using gdiplus and cocoa, so there shouldn't be issues.
You can alternatively use SvgImage's for your rounded borders, something that's offered in [fltk-theme](https://github.com/fltk-rs/fltk-theme) with the SvgBased scheme.

2- RadioButton (not RadioRoundButton).

**Reply by 4JX** [reply]
> 2- RadioButton (not RadioRoundButton).

> Creates a radio button. Radio meaning only one can be toggled in the same group.

To confirm, to properly use it I should add howver many `RadioButton`s I need to be unique from eachother to a [`Group`](https://docs.rs/fltk/latest/fltk/group/struct.Group.html)?

**Reply by MoAlyousef** [reply]
Group or any GroupExt implementing widget. i.e. if you have 2 RadioButton's in a window, only one can be toggled at a time since Window implements GroupExt.

**Reply by 4JX** [reply]
Good to know, thanks!

**Reply by MoAlyousef** [reply]
No problem:)
